pkgs/tools/X11/opentabletdriver/update.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env nix-shell
-#!nix-shell -i bash -p curl gnused jq common-updater-scripts nuget-to-nix dotnet-sdk_6 dotnet-sdk_5
+#!nix-shell -i bash -p curl gnused jq common-updater-scripts
```

Comment:
This seems to be missing `-I nixpkgs=../../../..`?
```suggestion
#!nix-shell -I nixpkgs=../../../.. -i bash -p curl gnused jq common-updater-scripts
```